typerforslack/settings.py

```diff
@@ -27,7 +27,8 @@
 # Database
 # https://docs.djangoproject.com/en/3.0/ref/settings/#databases
 
-LOCAL_URl = 'postgres://sanjay:typeit@localhost/typeforslack'
+LOCAL_URl = 'postgres://aravind:password@localhost/typeforslack'
```

Comment:
Let's change this to a common username like `admin`

Reply:
oh yes..need to be a standard one